HealthcareSystem/Backend/Service/NotificationSurveyAndFeedback/FeedbackService.cs

```diff
@@ -23,17 +23,13 @@ public FeedbackService(IFeedbackRepository feedbackRepository)
         }
         public void AddFeedback(Feedback feedback)
         {
-            if (feedback == null)
-                throw new BadRequestException("Please, write a comment to send feedback.");
             _feedbackRepository.AddFeedback(feedback);
         }
         public void PublishFeedback(int id)
         {
-            Feedback feedback = GetFeedbackById(id);
-            if (!feedback.IsAllowedToPublish)
-                throw new BadRequestException("Feedback is not allowed to publish.");
-            feedback.IsPublished = true;
-            _feedbackRepository.UpdateFeedback(feedback);
+            Feedback _feedback = GetFeedbackById(id);
+            _feedback.IsPublished = true;
+            _feedbackRepository.UpdateFeedback(_feedback);
```

Comment:
Our teaching assistant requires that try-catch be made in controllers layer such as FeedbackController (WebApp).  I suggest that you better familiarize with code, in case you don't see throwing exception and making try-catch blocks.

Reply:
I have already familiarized myself with code. However, there is something that I disagree with. I am not satisfied because you made exception handling only on your app, I think it is better to have that in our backend maybe. But if your product owner wants that in the controller layer. Alright then, but that is only good for your team I think, not for our organization.

Reply:
Our product owner thinks that all exceptions should exist in "higher" layers (such as Validation class in WebApp) because it will prevent that errors come to lower layers. That is, in the case of exception, some message will be returned to the user immediately instead of trying to write "something wrong" to the database. After the first sprint, we can consider your attitude about validation with the product owner.

Reply:
That is definitely a healthy discussion, and we should talk about it in the next meeting.